internal/datanode/channel_meta.go

```diff
@@ -111,13 +114,26 @@ type ChannelMeta struct {
 
 	metaService  *metaService
 	chunkManager storage.ChunkManager
+	workerPool   *concurrency.Pool
+}
+
+type lazyLoadTask struct {
```

Comment:
lazyLoadTask seems not to be used at all

Reply:
removed